charts/mgmt-agent/values.yaml

```diff
@@ -50,6 +50,16 @@ kubernetesCluster:
    name:
    # -- Kubernetes cluster namespace(s) to monitor. This can be a comma-separated list of namespaces or '*' to monitor all the namespaces
    namespace: '*'
+   # -- OCI namespace to push Kubernetes Monitoring metrics. The namespace should match the pattern '^[a-z][a-z0-9_]*[a-z0-9]$'. By default metrics will be pushed to 'mgmtagent_kubernetes_metrics'
+   monitoringNamespace:
```

Comment:
Should we move this and the other defined variables below this under mgmtagent ? instead of KubernetesCluster.

Reply:
monitoringNamespace and the four overrideAllowMetrics are part of cluster, so I believe it's under right section